src/modules/block.F90

```diff
@@ -477,7 +477,7 @@ module block
 
      !
      !         Flow variables.
-     !
+     ! x(0:ie,0:je,0:ke,3)
```

Comment:
remove this diff, the explanation of the x variable is listed above already. this line should just be an empty comment as it used to be